docs/src/filters/writing_custom_filters.md

```diff
@@ -206,14 +207,70 @@ First let's create the config for our static configuration:
 {{#include ../../../examples/quilkin-filter-example/src/main.rs:filter}}
 ```
 
+###### 4. Finally, update `GreetFilterFactory` to extract the greeting from the passed in configuration and forward it onto the `Greet` Filter.
```

Comment:
I really like this being back in - since it slowly introduces people to the config option, and then we can move into `deserialize`, which is the more complex topic. 👍🏻